### PR TITLE
Assignment

### PR DESCRIPTION
**Pull request by tavisca-vrai**
@apalsapure  please review my code

### CODE REVIEW

**Comment by tavisca-vrai** [comment]
@apalsapure  please review

**Comment by tavisca-vrai** [comment]
@apalsapure please review

File: 02.OperatorOverloading/OperatorOverloading/OperatorOverloading.DBL/Converter.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        {
+            int count=0;
+            if (finalCurrency == "USD")
+            {
+                string temp = finalCurrency;
+                finalCurrency = initialCurrency;
+                initialCurrency = finalCurrency;
+                count++;
+            }
+            StreamReader file = new StreamReader(@"D:\exchange rate.txt");
```
Comment (apalsapure) [inline 02.OperatorOverloading/OperatorOverloading/OperatorOverloading.DBL/Converter.cs:23]: Why this is not in App.config? Why not use File.ReadAllLines?

File: 02.OperatorOverloading/OperatorOverloading/OperatorOverloading.DBL/Converter.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+namespace OperatorOverloading.DBL
+{
+    public class CurrencyConverter : ICurrencyConverter
+    {
+        public double GetConversionRate(string initialCurrency, string finalCurrency)
+        {
+            int count=0;
+            if (finalCurrency == "USD")
```
Comment (apalsapure) [inline 02.OperatorOverloading/OperatorOverloading/OperatorOverloading.DBL/Converter.cs:16]: Why USD is hard coded?

**Pull request merged**